plugins/importexport/crossref/locale/en_US/locale.po

```diff
@@ -94,7 +97,7 @@ msgstr ""
 "\t\t<p>If a deposit has failed, please solve the problem and try to register the DOI again.</p>"
 
 msgid "plugins.importexport.crossref.action.export"
-msgstr "Download XML"
```

Comment:
When changing a locale key, make sure to mark all the other languages "fuzzy" (see e.g. https://til.hashrocket.com/posts/wcmc04nort-fuzzy-translation-key-merging-with-gettext) so that the translations can be reviewed.

Reply:
@asmecher for every other locale, should I add this? (already did, just wondering if that was what I had to do)

```
#, fuzzy
msgid "plugins.importexport.crossref.action.export"
msgstr "Tải xuống XML"
```

Reply:
Yes, that's right, thanks!